timescaledb/how-to-guides/migrate-data/import-csv.md

```diff
@@ -0,0 +1,95 @@
+# Import data into TimescaleDB from .csv
```

Comment:
I see some bugs related to highlight close to headers. Maybe we should have a short breath between the elements?
Check #53.
```suggestion
# Import data into TimescaleDB from .csv

```